### PR TITLE
Add standalone network used in docker standalone configurations

### PR DESCRIPTION
**Pull request by leighmcculloch**
### What
Add the standalone network that is used in the official Stellar Docker images to the built-in config.

### Why
The official Stellar Docker images ship with three networks, pubnet, testnet, and standalone. Standalone is useful for private testing and development. It'd be convenient if stc shipped with built-in support for the standalone network so that when testing with the docker images in standalone mode we didn't have to setup the network manually each time in stc config.

https://github.com/stellar/docker-stellar-core-horizon/blob/23e8df8/standalone/core/etc/stellar-core.cfg#L9

### CODE REVIEW

**Comment by stanford-scs** [comment]
Just a couple of questions:  Does the standalone network need to be in the default file, or should the docker image just include an `stc.conf` file?  It would be nice if the standalone target network only existed where it could actually be used, but I suppose there's not too much harm to including it otherwise.  Also, should the `standalone` target have a url, like http://127.0.0.1:some-port/?

**Comment by leighmcculloch** [comment]
I think the win with convenience is to build it in rather than require the file to be bundled with the quickstart image. Most people using the quickstart image don't check out its repo and so wouldn't have the file on disk anyway. Setting a URL as you suggest for Horizon is a good idea too, and port 8000 is the common port for testing Horizon locally with quickstart. I pushed 4418e69 adding it.

Do we need `native-asset` to be defined too?

File: config.go [hunk 1]
```diff
@@ -31,6 +31,10 @@ native-asset = XLM
 horizon = https://horizon-testnet.stellar.org/
 native-asset = TestXLM
 
+[net "standalone"]
+network-id = "Standalone Network ; February 2017"
+horizon = http://localhost:8000/
+
```
Comment (leighmcculloch) [inline config.go:37]:
```suggestion
[net "standalone"]
network-id = "Standalone Network ; February 2017"
horizon = http://localhost:8000/
native-asset = StandaloneXLM
```

**Comment by leighmcculloch** [comment]
I've added the native babe to the standalone network and this is ready for review.

**Review by stanford-scs — APPROVED** [review]

**Pull request merged**